### DISCUSSION TITLE
preflight:false still styling input

### DISCUSSION THREAD

**Discussion by LucasMonteiroDito** [tailwindlabs/tailwindcss#14645] [Help] [answered]
Hey guys. I'm kind new using tailwind and i have a question. When i'm using preflight as false, the generated css still have a stylization of some kind of inputs. We have a lib of components that is published at npm and we use this lib into another project, and this issue with preflight is making a conflict with the main project, that uses styled components.  Have i understood something wrong in the doc or this is a issue?

Here's some info about the config:

 tailwindcss: 3.4.13 
 
![image](https://github.com/user-attachments/assets/9e34678f-d931-4bd8-917d-46a5eb35335e)

generated css after bundle:
![image](https://github.com/user-attachments/assets/7e1d6fa7-d8f4-4921-ad33-b7ad4aef1915)

**Comment by wongjn** [comment, marked as answer]
That would be coming from the [`@tailwindcss/forms`](https://github.com/tailwindlabs/tailwindcss-forms) Tailwind plugin you have included.

**Reply by LucasMonteiroDito** [reply]
it was! Thanks for the help
